src/components/landing/BenefitsCarousel.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+// Import Swiper React components
+import { Swiper, SwiperSlide } from "swiper/react";
+
+// Import Swiper styles
+import "swiper/css";
+import "swiper/css/pagination";
+
+import "../../styles/landing/benefitsCarousel.css";
+
+// import required modules
+import { EffectCoverflow, Navigation, Pagination } from "swiper/modules";
+
+const BenefitsCarousel = ({ slides }) => {
+  const [_active, _setActive] = useState(0);
+  const [_activeIndex, setActiveIndex] = useState(0);
+
+  const [showPrevButton, setShowPrevButton] = useState(false);
+  const [showNextButton, setShowNextButton] = useState(true);
+
+  const handleSlideChange = (swiper) => {
+    setActiveIndex(swiper.activeIndex);
+    setShowPrevButton(!swiper.isBeginning);
+    setShowNextButton(!swiper.isEnd);
+  };
+
+  const colors = ["#EDFCE2", "#EAE2FC", "#FCF6E2", "#EDF2FE"];
+
+  return (
+    <div className="lg:flex justify-between w-full  items-center p-16  hidden relative">
+      <div
+        className="swip-prev slider-arrow relative  left-[12.5%] translate-x-[-50%]  z-40   borderLine rounded-full shadow-md p-4"
```

Comment:
```suggestion
        className="swip-prev slider-arrow relative left-[12.5%] -translate-x-1/2 z-40 select-none borderLine rounded-full shadow-md p-4"
```
nit: add `select-none` to stop highlighting text on the page when clicking the swiper arrow div multiple times